static/app-info.json

```diff
@@ -0,0 +1,14 @@
+{
+  "packages": [
+    {
+      "appName": "WARNING",
+      "androidPackageName": "counterfeit.warning",
+      "latestVersionCode": "0.0",
+      "appShortDescription": "You are using a counterfeit version of ReVanced Manager.  It is highly recommended to uninstall this app and uninstall all apps you installed using this. To learn more, visit https://counterfeits.revanced.app ",
+      "requireMicroG": false,
+      "latestVersionUrl": "https://dummydomain",
+      "icon": "https://counterfeits.revanced.app/red-alert-icon.png",
```

Comment:
Could replace with this.
![skull-sign-triangle-shape-icon](https://github.com/user-attachments/assets/38d3d85c-73eb-479f-98d7-39f7cf6ead76)